docs/experimental/plugin_system.rst

```diff
@@ -135,7 +135,7 @@ Full example:
 
 .. code-block:: python
 
-   from res.enkf import ErtScript
+   from ert import ErtScript
    from ert.shared.plugins.plugin_manager import hook_implementation
```

Comment:
Will this be another PR to align?

Reply:
This is already in the `ert/__init__.py`, but yes, I will create another PR to align this with semeio.